.github/workflows/stale-issues.yaml

```
@@ -0,0 +1,22 @@
+name: "Stale issues"
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  stale:
+    timeout-minutes: 1
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.ORG_GITHUB_TOKEN }}
```

Isn't `GITHUB_TOKEN` enough? `ORG_GITHUB_TOKEN` has more privileges than `GITHUB_TOKEN`.

```suggestion
        repo-token: ${{ secrets.GITHUB_TOKEN }}
```